src/GitReportCL/DateAndPathValidation.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+namespace GitReport.CLI
+{
+    class DateAndPathValidation                        
+    {
+        public bool ValidateGitDiffReportArgs(string[] arguments, GitReportArguments getArg)         
+        {
+            DateTime.TryParse(arguments[0], out var SinceDate);
+            DateTime.TryParse(arguments[1], out var BeforeDate);
+            getArg.DateSince = SinceDate;
+            getArg.DateBefore = BeforeDate;
+
+            if (Directory.Exists(arguments[2]))
+            {
+                getArg.GitPath = arguments[2];
+            }
+            else
+            {
+                getArg.GitPath = "no path found";
+            }
+
+            if (getArg.DateSince == DateTime.MinValue ||
```

Comment:
Code should relay on boolean value returned from TryParse, not on out parameter value